spec/factories/other_advice_method.rb

```diff
@@ -1,5 +1,6 @@
 FactoryGirl.define do
   factory :other_advice_method do
     name { Faker::Lorem.sentence }
+    sequence(:cy_name) { |n| "Dull gyngor arall #{n}" }
```

Comment:
Sure, no problem - was looking at Faker's support for Welsh. None. I'll revert to the same as `name`. :smile: